website/content/v1.7/kubernetes-guides/network/deploying-cilium.md

```diff
@@ -9,7 +9,7 @@ aliases:
 
 This documentation will outline installing Cilium CNI v1.14.0 on Talos in six different ways.
 Adhering to Talos principles we'll deploy Cilium with IPAM mode set to Kubernetes, and using the `cgroupv2` and `bpffs` mount that talos already provides.
-As Talos does not allow loading kernel modules by Kubernetes workloads, `SYS_MODULE` capability needs to be dropped from the Cilium default set of values, this override can be seen in the helm/cilium cli install commands.
+Due to Kubernetes [Pod Security Admission]({{< relref "../configuration/pod-security">}}), Talos does not allow loading kernel modules by Kubernetes workloads, `SYS_MODULE` capability needs to be dropped from the Cilium default set of values, this override can be seen in the helm/cilium cli install commands.
```

Comment:
this is incorrect, Talos doesn't allow loading kernel modules, but it has nothing to do with PSA.

Reply:
You are right. let me remove this.

I think just the last part is enough for people to not waste as much time as I did trying to figure out why the `cilium connectivity test` will fail right after install. I am still stuck with k8s <1.25 in most of my deployments and I am sure I am not the only one. We just need a link somewhere down the page.